com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -177,6 +178,7 @@ public UniversalRenderer(UniversalRendererData data) : base(data)
             // we inject the builtin passes in the before events.
             m_MainLightShadowCasterPass = new MainLightShadowCasterPass(RenderPassEvent.BeforeRenderingShadows);
             m_AdditionalLightsShadowCasterPass = new AdditionalLightsShadowCasterPass(RenderPassEvent.BeforeRenderingShadows);
+            this.stripShadowsOffVariants = true;
```

Comment:
This should be used to init the `forwardInitParams.additionalLightsAlwaysEnabled` parameter, so they stay synced up.